src/core/theme/zindexes.ts

```diff
@@ -1,3 +1,4 @@
 export const zindexes = {
```

Comment:
might be good idea to document these used z-indexes, if not already

Reply:
Yeah have to think about later, might be refactored also. This is inside theme so at this point only key-names should be something that need to be understand. Else just avoid using z-indexes or use 1, 2, 3 that are local and modular.